test/requirements/python-requirements.spec.js

```diff
@@ -0,0 +1,28 @@
+/*jshint node:true, mocha:true*/
+
+'use strict';
+const { spawnSync } = require('child_process');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+describe('python-test', function () {
+    it('Should find python 3', (done) => {
```

Comment:
If the test is synchronous, `done` is not needed.

Reply:
Also, "should" is generally lowercase when using mocha. Not a huge deal but might look a little funny.

Reply:
Will get rid of them.